src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -200,12 +273,20 @@
    (fn [{:keys [this props state]}]
      (let [new? (:new-registration? props)
            update? (:update-registration? props)]
-       [:div {:style style/thin-page-style}
+       [:div {:style {:minHeight 300 :paddingTop "1.5rem" :marginLeft "15rem" :marginRight "15rem"}}
```

Comment:
you should be able to set `:margin "0 15rem 0"`, but is that what you really want? this will be a problem at smaller window sizes.